### PR TITLE
Update MCP mappings to snapshot_nodoc_20161220

### PR DESCRIPTION
**Pull request by mezz**

### CODE REVIEW

File: src/main/java/net/minecraftforge/fml/common/asm/transformers/ItemStackTransformer.java [hunk 1]
```diff
@@ -81,7 +81,7 @@ else if (GETITEM_DESC.equals(m.desc))
 
         for (MethodNode m: classNode.methods)
         {
-            if (m.name.equals("forgeInit") || m.name.equals("func_190926_b")) continue;
+            if (m.name.equals("forgeInit") || m.name.equals("isEmpty") || m.name.equals("func_190926_b")) continue;
```
Comment (mezz) [inline src/main/java/net/minecraftforge/fml/common/asm/transformers/ItemStackTransformer.java]: Is this the right thing to do?

File: mdk/build.gradle [hunk 2]
```diff
@@ -71,7 +71,7 @@ processResources
         expand 'version':project.version, 'mcversion':project.minecraft.version
     }
         
-    // copy everything else, thats not the mcmod.info
+    // copy everything else except the mcmod.info
```
Comment (liach) [inline mdk/build.gradle:73]: You should fix the brace on line 61

Comment (mezz) [reply mdk/build.gradle:73]: ok

Comment (liach) [reply mdk/build.gradle:73]: Thanks a lot for fixing a minor issue that has existed since start of the use of gradle

**Pull request merged**